README.md

```diff
@@ -1,31 +1,72 @@
 # 1Password Connect Go SDK
 
-The 1Password Connect Go SDK provides access to the 1Password Connect API hosted on your infrastructure. The library is intended to be used by your applications, pipelines, and other automations to simplify accessing items stored in your 1Password vaults.
+[![Go Reference](https://pkg.go.dev/badge/github.com/1Password/connect-sdk-go.svg)](https://pkg.go.dev/github.com/1Password/connect-sdk-go)
+[![Go Report Card](https://goreportcard.com/badge/github.com/1Password/connect-sdk-go)](https://goreportcard.com/report/github.com/1Password/connect-sdk-go)
+[![Version](https://img.shields.io/github/release/1Password/connect-sdk-go.svg)](https://github.com/1Password/connect-sdk-go/releases/)
 
-## Installation
+The 1Password Connect Go SDK provides access to the 1Password Connect API, to facilitate communication with the Connect server hosted on your infrastructure. The library is intended to be used by your applications, pipelines, and other automations to simplify accessing items stored in your 1Password vaults.
 
+## Installation
+To download and install the 1Password Connect Go SDK, as well as its dependencies:
 ```sh
 go get github.com/1Password/connect-sdk-go
 ```
```

Comment:
Let's also include how to import the library. For example:

Then, import it using:

import (
    "github.com/1Password/connect-sdk-go/connect"
)